brainstormforce/ultimate-addons-for-gutenberg
# Improvement: Team - Added span and div tag support to title.

### Description

- https://app.startinfinity.com/b/Mi5zu3VRVzk/M7Dpq9yR4Pe/0eb190bb-a1aa-4261-968f-d8ef662c6b42?view=0ce18087-e6b2-4a91-b54c-c386ce1f811c&t=attributes

### Types of changes




- New feature (non-breaking change which adds functionality)

### How has this been tested?

- Drag and drop team block.
- Change title tag to div span.
- Add a styling color typography it should be applied on the backend as well as the frontend.

### Checklist:
